docs/sample.yml

```diff
@@ -82,45 +109,14 @@ paths:
               schema:
                 type: array
                 items:
-                  $ref: "#/components/schemas/Course"
+                  $ref: "#/components/schemas"
```

Comment:
How about this instead of `$ref: "#/components/schemas"`?
```
properties:
  id: 
    type: string
    description: this is a description of column
  status: 
    type: string
    description: |
      User status<br>
      Acceptable values are:
      - wait list - who waits for a next course
      - applied - who applied for a course but not paid yet
      - registered - who applied for a course and paid
      - resign - who quit a course
```